assets/tests/components/routeLadder.test.tsx

```diff
@@ -262,6 +262,10 @@ describe("routeLadder", () => {
       />
     )
 
+    expect(
+      screen.getByRole("button", { name: "28 Route Options" })
```

Comment:
Curiosity: Where does the `28` here come from? I see how `Route Options` gets into the button's name, and I theoretically see why `28` is appropriate, but I can't figure out how they get stitched together.

Reply:
Hmm, would this be a better assertion?
```suggestion
      screen.getByRole("button", { name: "${route.name} Route Options" })
```

Reply:
Ooooh does `aria-labelledby=[routePillId, routeOptionsToggleId]` translate the `[routePillId, routeOptionsToggleId]` into their `name`'s `["28", "Route Options"]` and then concatenate?

Reply:
> Hmm, would this be a better assertion?

Heh no I like the assertion as it is - I just didn't understand the code mechanism that made the test pass 😅

Reply:
> Ooooh does `aria-labelledby=[routePillId, routeOptionsToggleId]` translate the `[routePillId, routeOptionsToggleId]` into their `name`'s `["28", "Route Options"]` and then concatenate?

That's essentially correct AFAIK, sorry about the comment before, I was misremembering things

Reply:
> Ooooh does `aria-labelledby=[routePillId, routeOptionsToggleId]` translate the `[routePillId, routeOptionsToggleId]` into their `name`'s `["28", "Route Options"]` and then concatenate?

Just to be clear, we create a [space-separated id reference list](https://developer.mozilla.org/en-US/docs/Web/Accessibility/ARIA/Attributes/aria-labelledby), and then screen readers and accessibility apis are responsible for doing the concatenation, this doesn't introduce any clones of the data from the DOM, and instead acts like pointers